manifests/t/the-sz/Seaside/1.37/the-sz.Seaside.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.Seaside
+PackageVersion: 1.37
+PackageLocale: en-US
+Publisher: the sz development
+PackageName: Seaside

```

Comment:
```suggestion
PackageName: Seaside Multi Skype Launcher
```

Reply:
The yaml file is auto-generated based on my web page database. Is it really required this change?

"ShortDescription" contains the description "Start multiple Skype sessions"

Reply:
Yes and no since its looking for `Seaside *`, but its better to be on the safe side.